app/config/core.php

```diff
@@ -171,8 +171,12 @@
 /**
  * Session time out time (in seconds).
  * Actual value depends on 'Security.level' setting.
+ * When using CakePHP 1.3.x:
+ *   - cookie expiration (in seconds) = "this value" * Security::inactiveMins() * 60
+ *   - session timeout (in seconds) = "this value" * Security::inactiveMins()
+ * NOTE: With CakePHP 2.x, this value is in minutes.  And security level won't affect timeout
  */
-	Configure::write('Session.timeout', '1200');
+	Configure::write('Session.timeout', '20');
```

Comment:
So the new timeout is 20mins? It might be a little bit short. As some of the forms have quite a few questions and students have to do one for each team member. So it may take more than 20mins to finish. We can increase it to 2 to 4 hours to be safe.

Reply:
Is iPeer still using CakePHP 1.3.x in production? If so, assigning 20 to`Session.timeout` will set the session timeout to 20 * 300 seconds = 6000 seconds = 100 minutes.

Reply:
Let's set it to 3 hours as discussed on slack. Thanks.

Reply:
Revised the value to 36.  That will set the session timeout to (36 * 300) seconds = 10800 seconds = 180 minutes.